### ISSUE TITLE
pip installation fails: "nrt.h: No such file or directory"

### ISSUE THREAD

**Issue by gmarkall** [numba/numba#1211]
cc @izaid 

From the end of the log (on Linux, also fails on OS X in a similar manner): 

```
    building 'numba.runtime._nrt_python' extension
    C compiler: gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC
    
    creating build/temp.linux-x86_64-2.7/numba/runtime
    compile options: '-Inumba -I/home/gmarkall/miniconda3/envs/piptest/lib/python2.7/site-packages/numpy/core/include -I/home/gmarkall/miniconda3/envs/piptest/include/python2.7 -c'
    gcc: numba/runtime/_nrt_python.c
    numba/runtime/_nrt_python.c:2:17: fatal error: nrt.h: No such file or directory
     #include "nrt.h"
                     ^
    compilation terminated.
    numba/runtime/_nrt_python.c:2:17: fatal error: nrt.h: No such file or directory
     #include "nrt.h"
                     ^
    compilation terminated.
    error: Command "gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Inumba -I/home/gmarkall/miniconda3/envs/piptest/lib/python2.7/site-packages/numpy/core/include -I/home/gmarkall/miniconda3/envs/piptest/include/python2.7 -c numba/runtime/_nrt_python.c -o build/temp.linux-x86_64-2.7/numba/runtime/_nrt_python.o" failed with exit status 1
    
    ----------------------------------------
Command "/home/gmarkall/miniconda3/envs/piptest/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip-build-uFzAIr/numba/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-Ni6GUY-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-uFzAIr/numba
```

**Comment by gmarkall** [comment]
Further investigation suggests that nrt.h is not part of the source distribution:

```
$ cd /tmp/pip-build-BoFGJQ/numba/numba/runtime
$ ls
atomicops.py  __init__.py  nrt.c  nrt.py  _nrt_python.c
```

**Comment by gmarkall** [comment]
Fix proposed in #1212.
